### PR TITLE
Use integer division in examples when using transform.resize

### PR DESCRIPTION
**Pull request by Alex-Linhares**
solves "ValueError: Integer argument required but received (817.8, 516.8, 3), check inputs."

## Checklist

- [ ] [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
- [ ] Gallery example in `./doc/examples` (new features only)
- [ ] Benchmark in `./benchmarks`, if your changes aren't covered by an
  existing benchmark
- [ ] Unit tests
- [ ] Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)

## For reviewers

- [ ] Check that the PR title is short, concise, and will make sense 1 year
  later.
- [ ] Check that new functions are imported in corresponding `__init__.py`.
- [ ] Check that new features, API changes, and deprecations are mentioned in
      `doc/release/release_dev.rst`.
- [ ] Consider backporting the PR with `@meeseeksdev backport to v0.14.x`

### CODE REVIEW

File: doc/examples/transform/plot_rescale.py [hunk 1]
```diff
@@ -27,8 +27,8 @@
 
 image = color.rgb2gray(data.astronaut())
 
-image_rescaled = rescale(image, 1.0 / 4.0, anti_aliasing=False)
-image_resized = resize(image, (image.shape[0] / 4, image.shape[1] / 4),
+image_rescaled = rescale(image, 1.0 // 4.0, anti_aliasing=False)
```
Comment (jni) [inline doc/examples/transform/plot_rescale.py]: I'm ok with the line below, but this one is surely incorrect?

```python
In [1]: 1.0 // 4.0
Out[1]: 0.0
```

Comment (Alex-Linhares) [reply doc/examples/transform/plot_rescale.py]: You're right; I fell into an error in resize; and didn't quite realize that we also had rescale.  My mistake!

Comment (jni) [reply doc/examples/transform/plot_rescale.py]:
```suggestion
image_rescaled = rescale(image, 0.25, anti_aliasing=False)
```

**Comment by emmanuelle** [comment]
Hi @Alex-Linhares would you have the time to make the change requested by @jni so that we can merge your PR?

**Comment by hmaarrfk** [comment]
@Alex-Linhares did you want to make the suggestion @jni made? That is the last outstanding thing in the PR.

Pending that, this PR has my approval.

**Comment by sciunto** [comment]
Replaced by #4032

**Pull request closed without merging**